typescript/agentkit/src/wallet-providers/cdpWalletProvider.ts

```diff
@@ -107,6 +124,8 @@ export class CdpWalletProvider extends EvmWalletProvider {
       chain: NETWORK_ID_TO_VIEM_CHAIN[config.network!.networkId!],
       transport: http(),
     });
+    this.#gasLimitMultiplier = Math.max(config.gas?.gasLimitMultiplier ?? 1, 1);
+    this.#feePerGasMultiplier = Math.max(config.gas?.feePerGasMultiplier ?? 1, 1);
```

Comment:
Viem default to 1.2x multiplier (see: [here](https://github.com/wevm/viem/blob/main/src/actions/public/estimateFeesPerGas.ts#L120))

Should we do the same?

Reply:
Interesting. I'm fine with adding a default value for the gasLimitMultiplier. That won't cost users anything extra, it just gives their transactions a little buffer room.